### DISCUSSION TITLE
Query regarding Algorithm used

### DISCUSSION THREAD

**Discussion by AnjaliSingh04** [Authenticator-Extension/Authenticator#840] [Q&A] [answered]
Hi, I was curious to understand whether these browser extensions (Google Authenticator and Authenticator) supports the following: OATH TOTP algorithm (specified in RFC 6238) https://datatracker.ietf.org/doc/html/rfc6238

**Comment by mymindstorm** [comment, marked as answer]
We support HOTP and TOTP.

**Reply by AnjaliSingh04** [reply]
Thanks for getting back on this. Do any of your Salesforce customers use these apps? Please confirm.

**Comment by Sneezry** [comment]
Yes, it works with RFC 6238.
https://authenticator.cc/docs/en/overview

mymindstorm ***@***.***>于2022年1月18日 周二下午11:07写道：
